kitsune/sumo/static/sumo/scss/components/_messages.scss

```diff
@@ -0,0 +1,87 @@
+@use '../config' as c;
+@use 'protocol/css/includes/lib' as p;
+
+.inbox-table {
+    display: grid;
```

Comment:
I understand that the html will be updated to use a list so css will be changed but fwiw I would stick with flexbox for this and I wouldn't use grid

Reply:
Converted.

Reply:
I'm definitely not a CSS expert, so I'm genuinely wondering why you'd recommend against using `grid`? Is there a drawback to using it?